Gems/RemoteTools/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+
+set(gem_path ${CMAKE_CURRENT_LIST_DIR})
+set(gem_json ${gem_path}/gem.json)
+o3de_restricted_path(${gem_json} gem_restricted_path gem_parent_relative_path)
+
+o3de_pal_dir(pal_dir ${CMAKE_CURRENT_LIST_DIR}/Platform/${PAL_PLATFORM_NAME} "${gem_restricted_path}" "${gem_path}" "${gem_parent_relative_path}")
+
+# Now that we have the platform abstraction layer (PAL) folder for this folder, thats where we will find the
+# project cmake for this platform.
+include(${pal_dir}/${PAL_PLATFORM_NAME_LOWERCASE}_gem.cmake)
```

Comment:
FYI You can nuke this include and the top level `Gems/RemoteTools/Platform` directory.

It is an unnecessary PAL directory and cuts 10 files from the gem